### PR TITLE
adds functionality to remove passport data points

### PR DESCRIPTION
**Pull request by aminah-io**

### CODE REVIEW

File: platforms/src/GtcStaking/__tests__/communityStaking.test.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (nutrina) [inline platforms/src/GtcStaking/__tests__/communityStaking.test.ts:1]: Do we really need this eslint-disable?

**Comment by nutrina** [comment]
I have this error:
image

The problem is that when getting the challenge credential, we must send a type ... however do we need this in the IAM ????

File: platforms/src/GtcStaking/__tests__/communityStaking.test.ts (hunk 1 above)
Comment (aminah-io) [reply platforms/src/GtcStaking/__tests__/communityStaking.test.ts:1]: I suppose we don't, but I wasn't able to commit my changes until I added it. Is there another flag we can use to bypass eslint for now?

Comment (nutrina) [reply platforms/src/GtcStaking/__tests__/communityStaking.test.ts:1]: We should not bypass eslint, but rather fix the linter errors.

File: app/components/GenericPlatform.tsx [hunk 2]
```diff
@@ -127,39 +127,45 @@ export const GenericPlatform = ({ platFormGroupSpec, platform }: PlatformProps):
...
-        signer as { signMessage: (message: string) => Promise<string> }
-      );
-      // because we provided a types array in the params we expect to receive a
-      // credentials array in the response...
-      const vcs =
-        verified.credentials
-          ?.map((cred: CredentialResponseBody): Stamp | undefined => {
-            if (!cred.error) {
+      // This array will contain all providers that new validated VCs
+      let vcs: Stamp[] = [];
```
Comment (schultztimothy) [inline app/components/GenericPlatform.tsx:131]: Nit: might be a smidge cleaner to keep `.map/.filter` https://github.com/gitcoinco/passport/pull/728/files#diff-76a78c07ce4bb2a6bbd9387f6e57db7df025cb43927cb966f5b1eca732c9d097L155 methods as opposed to using let here

**Review by schultztimothy — APPROVED** [review]
Works great 🚀 ! I left one little comment but LGTM

File: app/components/GenericPlatform.tsx (hunk 2 above)
Comment (aminah-io) [reply app/components/GenericPlatform.tsx:131]: I totally agree! The vcs weren't properly filtering with map and filter though, so we went with the for loop.

**Pull request merged**